default.yml

```diff
@@ -18,15 +18,42 @@
   vars:
     network_ipv6_enable: true
     ssh_allow_root_with_key: true
+    ssh_allow_tcp_forwarding: true
+    ssh_allow_agent_forwarding: true
+    ssh_server_permit_environment_vars: 'PWD'
+    ssh_client_alive_interval: 100
+    ssh_client_alive_count: 10
     ssh_client_password_login: true
     ssh_client_cbc_required: true
-    ssh_server_weak_hmac: true
     ssh_client_weak_kex: true
+    ssh_challengeresponseauthentication: true
+    ssh_compression: 'yes'
```

Comment:
`ssh_compression: 'yes'` currently only evaluates to `True` because any non-empty string evaluates to `True`. `ssh_compression: 'no'` would too therefore. To catch this, I suggested to use the `bool` filter below.

For consistency you could change this to `true`.